src/views/GitDescription.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <description-layout />
+</template>
+
+<script lang="ts">
+import { Component, Vue } from "vue-property-decorator";
+
+import DescriptionLayout from "@/components/Description/DescriptionLayout.vue";
```

Comment:
@JuanJosePG  description* Please review the inport or the name from the filters, cause in my pc there are problems with the folders name